gufe/tokenization.py

```diff
@@ -539,8 +571,13 @@ def from_dict(dct) -> GufeTokenizable:
 def _from_dict(dct: Dict) -> GufeTokenizable:
     module = dct.pop('__module__')
     qualname = dct.pop('__qualname__')
+    if ':version:' in dct:
+        version = dct.pop(':version:')
```

Comment:
isn't this just `dct.pop(':version:', 1)`?

Reply:
So, I literally looked that up because I thought there should be such a thing, but didn't see it in the docs. I think I accidentally hit the docs for `list.pop`.

Yes, this is what it should have been. Fixed.